tests/unit/test_charm.py

```diff
@@ -249,6 +249,22 @@ def test_configuration_reload_read_timeout(self, trigger_configuration_reload, *
         self.harness.evaluate_status()
         self.assertIsInstance(self.harness.model.unit.status, MaintenanceStatus)
 
+    def test_check_disk_space(self):
+        test_cases = [
+            (1e10, ActiveStatus),
+            (1e8, BlockedStatus),
```

Comment:
- Let's use `1024**3` and `1024**3 + 1`.
- Consider using `unittest.subTest` in addition to the for loop below.